org.eclipse.jdt.ls.product/scripts/jdtls

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/env python3
```

Comment:
I think [that approach](https://stackoverflow.com/a/3592208/5088458) should be used so that it is possible to use it on Windows and Linux by just typing `jdtls` in the command line.